controllers/managedocs_controller.go

```diff
@@ -1502,26 +1502,27 @@ func (r *ManagedOCSReconciler) reconcileEgressNetworkPolicy() error {
 			}
 
 			// Get the machine cidr
-			vpcCIDR, ok := awsConfigMap.Data[utils.CIDRKey]
-			if !ok {
+			vpcCIDRs := awsConfigMap.Data
+			if vpcCIDRs == nil {
 				return fmt.Errorf("Unable to determine machine CIDR from AWS ConfigMap")
 			}
 			// Allow egress traffic to that cidr range
-			vpcEgressRule := openshiftv1.EgressNetworkPolicyRule{
-				Type: openshiftv1.EgressNetworkPolicyRuleAllow,
-				To: openshiftv1.EgressNetworkPolicyPeer{
-					CIDRSelector: vpcCIDR,
-				},
+			var vpcEgressRules []openshiftv1.EgressNetworkPolicyRule
+			for _, cidr := range vpcCIDRs {
```

Comment:
why is `vpcCIDRs` a map if we never use its key? Could we make it a []string instead?

Reply:
`vpcCIDRs` is the data retrieved from the configMap, which is of type map[string]string, so I think converting it to an array/slice of strings and then using it to update egressRules won't be a good idea